### DISCUSSION TITLE
Facing issues trying to generate unit test for Azure synapse DB

### DISCUSSION THREAD

**Discussion by shishirdk** [sanderstad/PStSQLtTestGenerator#50] [General]
When trying the generate unit test for Procs on Azure synapse.
[Invoke-PSTGTestGenerator] Something went wrong creating the procedure existence tests | The following exception occurred while trying to enumerate the collection: "This object is not supported on Datawarehouse databases.".

Is there any compatible version for Azure synapse? 
Is there any way we can have the code coverage pulled to be integrated into build pipeline?

**Comment by sanderstad** [comment]
I have not tried to create these items directly from a Synapse database.
I will look into that
